.travis.yml

```diff
@@ -16,6 +16,7 @@ cache:
 install:
 - npm --production=false install
 - npm --production=false update
+- cd node_modules/scratch-vm && npm --production=false install && npm run build && cd -
```

Comment:
Note to revert this before merging

Reply:
Oops! I made the PR against the wrong branch. This one was only to be used for testing.